lib/livebook/hubs/team_client.ex

```diff
@@ -66,20 +66,32 @@ defmodule Livebook.Hubs.TeamClient do
   ## GenServer callbacks
 
   @impl true
-  def init(%Team{} = team) do
+  def init(%Team{offline: nil} = team) do
+    derived_keys = Teams.derive_keys(team.teams_key)
+
     headers = [
-      {"x-user", to_string(team.user_id)},
-      {"x-org", to_string(team.org_id)},
-      {"x-org-key", to_string(team.org_key_id)},
-      {"x-session-token", team.session_token}
+      {"x-user", to_string(team.user_id || 0)},
+      {"x-org", to_string(team.org_id || 0)},
+      {"x-org-key", to_string(team.org_key_id || 0)},
+      {"x-session-token", to_string(team.session_token)}
```

Comment:
Instead of adding those keys here, what do you think about:

1. Add `@enforce_keys [:user_id, :org_id, :org_key_id, :session_token, ...]` and all other required fields before the "schema" definition, so we make sure those are always set

2. Then, when we create the offline teams hub, we set those values to 0, removing the need for conditional here.